### PR TITLE
Refactored the CSS from a subdirectory to a global CSS file

### LINKED ISSUE #2514: Refactor CSS files in src/screens/OrganizationEvents
# Refactor CSS files in src/screens/OrganizationEvents

To improve the UI/UX of Talawa-Admin we have been working on:
1. making the application more suitable to color blind users
2. streamlining all CSS into a single global file

## Additional information:
1. The foundational work for this was completed in this PR:
    - https://github.com/PalisadoesFoundation/talawa-admin/pull/2466
2. The desired single CSS file is:
    - `src/style/app.module.css`

## The challenge:
The goal is to convert the CSS file in this subdirectory and all the components related to this screen to use this new design pattern. This means that:
1. All CSS files in this subdirectory and all the CSS files in related components must be merged into the global CSS file using pre-existing CSS.
2. All CSS files in this subdirectory and all the CSS files in related components must be deleted after the merge.
3. The remaining relevant files in this subdirectory must only reference the global CSS file.
4. All the CSS files in the related components of this subdirectory must adhere to the same guidelines.

### PR DESCRIPTION
**Pull request by harshk-89**
**What kind of change does this PR introduce?**

feature
**Issue Number:**

Fixes #2514 

**Did you add tests for your changes?**

yes

**Snapshots/Videos:**

WhatsApp Image 2024-12-29 at 05 44 56_6004fb8a

**If relevant, did you update the documentation?**

No
**Summary**

The motivation for this change is to improve styling consistency and maintainability by migrating CSS from OrganizationEvents to app.module.css. This resolves issues of style duplication and promotes a unified design. More details can be found in issue #2514(https://github.com/PalisadoesFoundation/talawa-admin/issues/2514).

**Does this PR introduce a breaking change?**

NO, It just moving CSS from a subdirectory to a global file.

**Other information**

This pull request migrates the CSS from OrganizationEvents to the global CSS file app.module.css. In this pull request, I attempted to use the existing CSS from app.module.css in the .tsx files within the OrganizationEvents directory. If I added new CSS, I aimed to utilize the existing colors from app.module.css to maintain design consistency.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

yes

## Summary by CodeRabbit

- **Style Changes**
	- Updated styling for the Organization Events screen.
	- Introduced new CSS classes for event-related components.
	- Migrated styles from a dedicated module to a centralized application stylesheet.

- **Refactor**
	- Consolidated CSS styling approach with updated class names.
	- Updated import paths for styling resources.

### CODE REVIEW

File: src/style/app.module.css [hunk 1]
```diff
@@ -3485,88 +3485,6 @@ button[data-testid='createPostBtn'] {
...
-}
-
-.recurrenceDayButton {
-  width: 33px;
-  height: 33px;
-  border: 1px solid var(--bs-gray);
-  cursor: pointer;
-  transition: background-color 0.3s;
-  display: inline-flex;
-  justify-content: center;
```
Comment (palisadoes) [inline src/style/app.module.css:3515]: 1. Why was this section deleted? 
1. Do references to these elements still exist in the code base?

**Review by palisadoes — CHANGES_REQUESTED** [review]

**Comment by palisadoes** [comment]
Please get code rabbit to approve your PR

**Comment by palisadoes** [comment]
Please fix the conflicting file

**Comment by harshk-89** [comment]
> Please fix the conflicting file

Actually, there are two issues addressing the same problem: issue #2888 (https://github.com/PalisadoesFoundation/talawa-admin/issues/2888) and issue #2514 (https://github.com/PalisadoesFoundation/talawa-admin/issues/2514). The pull request for issue #2888 has already been merged. Should I override the changes while resolving the merge conflict?

**Comment by palisadoes** [comment]
1. We should close this PR and the issue. It doesn't make sense duplicating the work. 
2. We try not to have duplicate issues, we made an error here

File: src/style/app.module.css
[the remark this answers is not in the document]
Comment (harshk-89) [reply src/style/app.module.css:1]: > Why did you removed these css ? Weren't they being used in components ?

i have fixed it,
but there are two issues addressing the same problem: issue https://github.com/PalisadoesFoundation/talawa-admin/issues/2888 (https://github.com/PalisadoesFoundation/talawa-admin/issues/2888) and issue https://github.com/PalisadoesFoundation/talawa-admin/issues/2514 (https://github.com/PalisadoesFoundation/talawa-admin/issues/2514). The pull request for issue https://github.com/PalisadoesFoundation/talawa-admin/issues/2888 has already been merged this pr is of no use hard luck

**Pull request closed without merging**